blog/_src/posts/2016-07-12-tutorial-zero-to-sixty-in-racket.scrbl

```diff
@@ -0,0 +1,543 @@
+#lang scribble/manual
+
+@; https://yanniss.github.io/law.html
+@; @hyperlink["http://www.ccs.neu.edu/racket/pubs/dls06-tf.pdf"]{scripts to programs}
+
+Title: Tutorial: Zero to Sixty in Racket
+Date: 2016-07-12T01:29:11
+Tags: Racket, tutorial, by Ben Greenman
+
+@(require scribble/example
+          scribble/racket
+          "../../utils/utils.rkt"
+          (for-syntax racket/base))
+@(define ev (make-base-eval))
+@(define SAMPLE "heart-of-darkness.txt")
+@(define TODOURL "http://www.google.com") @; placeholder for real urls
+
+
+@; -----------------------------------------------------------------------------
+
+Racket has excellent tools for helping scripts evolve into stable and useful programs.
+This post is about the why and how of the evolution,
+ and describes how various Racket features combine to solve everyday programs.
+
+<!-- more -->
+
+Why should anyone use Racket?
+There are two reasons:
+@itemlist[#:style 'ordered
+  @item{
+    You have a problem that can only be solved with Racket's unique
+     language-building tools
+     @; @cite[] Rosette
+     @; @cite[] Pollen
+     @; @cite[] Dotty
+  }
+  @item{
+    Racket is a nice language to program in.
+  }
+]
+
+Racket is nice for many, many reasons.
+It has lexical scoping.
+It has parentheses; you don't need to learn mysterious precendence rules
+  @; better to have a messy conditional
+  to parse @code{1 + 2 * 3}.
+When you first open the @hyperlink[TODOURL]{Dr.Racket} editor, there are no
+ distractions with project managers and classpath settings.
+The list goes on.
```

Comment:
I think you could skip those two first paragraphs and go straight to the point you want to illustrate in this post.